src/Nox.Ui.Blazor.Lib/Components/Generic/SelectList.razor.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Microsoft.AspNetCore.Components;
+using Microsoft.VisualBasic;
+using MudBlazor.Charts;
+using Nox.Ui.Blazor.Lib.Models;
+
+namespace Nox.Ui.Blazor.Lib.Components.Generic;
+
+public partial class SelectList : ComponentBase
```

Comment:
Maybe I'd rename it to SelectMany. @rochar what do you think?

Reply:
maybe MultiSelect?

SelectMany is more about flatten a lists of lists